circle.yml

```diff
@@ -6,16 +6,30 @@ machine:
   environment:
     ADMINS: Olivier Le Floch <olefloch@work4labs.com>
     SECRET_KEY: EamJrjviiW13ue8PEamJrjviiW13ue8P
+    MONGO_VERSION: 3.4.9
   python:
     version: 2.7.11
 
+checkout:
+  pre:
+    - git clone https://github.com/Work4Labs/circleci_toolkit.git ~/circleci_toolkit
+
 dependencies:
+  pre:
+    - ~/circleci_toolkit/bin/setup_mongo.sh
   override:
     - pip install --upgrade pip # To work around a pip 6.0.3 bug that fails to parse the setuptools version string
     - pip install --upgrade distribute # To work around mysql-python / virtualenv --distribute incompatibility
```

Comment:
I think these should be part of the toolkit features

Reply:
As discussed in another PR, opened FUP https://work4labs.atlassian.net/browse/BAT-7851
Is that good for now?